core/unit_test/CMakeLists.txt

```diff
@@ -1082,6 +1082,7 @@ endif()
     UnitTestMainInit.cpp
     tools/TestEventCorrectness.cpp
     tools/TestWithoutInitializing.cpp
+    tools/TestKernelNames.cpp
```

Comment:
Should you place it better in the list ? It seems these lists are usually given in lexicographic order.

Reply:
Didn't really feel like ordering the whole list and would rather not do it there